presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -259,6 +260,19 @@ public int getMaxQueryRunningTaskCount()
         return maxQueryRunningTaskCount;
     }
 
+    @Config("experimental.max-total-running-task-count-to-halt-scheduling")
+    @ConfigDescription("Keep queries in the queue if total task count exceeds experimental.max-total-running-task-count-to-halt-scheduling")
```

Comment:
nit: I suggest to not repeat the property name in the description: `Keep queries in the queue if total task count exceeds this threshold`